cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -1452,6 +1454,22 @@ func TestStaticAutoscalerRunOnceWithUnselectedNodeGroups(t *testing.T) {
 func TestStaticAutoscalerRunOnceWithBypassedSchedulers(t *testing.T) {
 	bypassedScheduler := "bypassed-scheduler"
 	nonBypassedScheduler := "non-bypassed-scheduler"
+
+	tmpDir, err := os.MkdirTemp("", "scheduler-configs")
```

Comment:
This seems to copy-paste logic used in schedulerbased_test.go, can we move it to a single utility function (for example in config/test/test_utils.go)?

Reply:
I restored this unit test.